packages/core/contracts/Escrow.sol

```diff
@@ -182,8 +182,20 @@ contract Escrow is IEscrow, ReentrancyGuard {
     }
 
     /**
-     * @dev Bulk payout workers
-     * Should fail if any of the transaction is failing.
+     * @notice Bulk payout workers
+     * Escrow needs to be complted / cancelled, so that it can be paid out.
```

Comment:
Add **@dev tag** from L186 to L193 because if you run `solc --devdoc --pretty-json Escrow.sol`  you won't have the expected file.